### PR TITLE
added spellcheck

### PR DESCRIPTION
**Pull request by Renetnielsen**
## References

* #0000

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,18 +13,18 @@
...
-*Energinet is not an IT company, and therefore we do not sell systems, services, or operate other TSO's. Our core business is clear, and it must be maintained, but we can contribute to the acceleration of change inside and outside the country, through the methods and tools we use – in this movement we see open source as an important tool.*
+*Energinet is not an IT company, and therefore we do not sell systems, services, or operate other TSOs. Our core business is clear, and it must be maintained, but we can contribute to the acceleration of change inside and outside the country, through the methods and tools we use – in this movement we see open source as an important tool.*
 
-*If we can accelerate and disseminate our green digital solutions, we see an opportunity to accelerate the green transition and increase socio-economic value creation."*
+*If we can accelerate and disseminate our green digital solutions, we see an opportunity to accelerate the green transition and increase socioeconomic value creation."*
```
Comment (ASQ-EN) [inline README.md:27]: socio economic

**Review by ASQ-EN — APPROVED** [review]
minor typo comment

File: README.md (hunk 1 above)
Comment (Renetnielsen) [reply README.md:27]: Will fix that

Comment (ASQ-EN) [reply README.md]: Please merge :)

**Pull request merged**